postfix/tests/common.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import getpass
+
+from datadog_checks.dev._env import deserialize_data, get_env_vars
```

Comment:
We should move those functions to a public module. They should be present in `env` actually, maybe add a `__all__` or something.